internal/pkg/runtime/engine/apptainer/container_linux.go

```diff
@@ -1831,6 +1840,19 @@ func (c *container) addBindsMount(system *mount.System) error {
 	return nil
 }
 
+func (c *container) copyHostLocaltime(localtimePath string) error {
+	if c.engine.EngineConfig.GetSessionLayer() == apptainer.OverlayLayer {
+		content, err := os.ReadFile(localtimePath)
+		if err != nil {
+			return fmt.Errorf("while reading host %s, err: %s", localtimePath, err)
```

Comment:
Would be good to ignore the error when `/etc/localtime` doesn't exist on the host filesystem:

```suggestion
            if !errors.Is(err, os.ErrNotExist) {
			    return fmt.Errorf("while reading host %s, err: %s", localtimePath, err)
			}
```

Reply:
thanks! I have updated the code.